drivers/net/wireless/mediatek/mt76/mt7921/main.c

```diff
@@ -295,12 +247,7 @@ static int mt7921_add_interface(struct ieee80211_hw *hw,
 		goto out;
 	}
 
-	idx = get_omac_idx(vif->type, phy->omac_mask);
-	if (idx < 0) {
-		ret = -ENOSPC;
-		goto out;
-	}
-	mvif->mt76.omac_idx = idx;
+	mvif->mt76.omac_idx = mvif->mt76.idx;
```

Comment:
@objelf have you checked the possible value for omac_idx according to the irc discussion?